direct/common/subsample.py

```diff
@@ -38,8 +36,8 @@ class BaseMaskFunc:
 
     def __init__(
         self,
-        accelerations: Optional[Tuple[Number, ...]],
-        center_fractions: Optional[Tuple[float, ...]] = None,
+        accelerations: Sized,
+        center_fractions: Sized = None,
```

Comment:
It can be typed. The error will raise in lines 57 and 60 if accelerations and/or center_fractions is not a list, e.g., accelerations = 4. This can be fixed by turning a single input into a list always.